src/itemset.js

```diff
@@ -96,6 +97,12 @@ define([
                     return value;
                 },
                 validationFunc: function (mug) {
+                    if (!mug.options.lookupTablesEnabled) {
+                        return "You no longer have access to Lookup Tables in your application. " +
+                            "Lookup Tables are available on the Standard plan and higher.\n" +
+                            "Before you can make a new version of your application, " +
+                            "you must [change your subscription](" + changeSubscriptionLink + ") or delete this question.";
```

Comment:
https://github.com/dimagi/Vellum/commit/b0be04a0ddadd158698bd49ffc68c6f39c362f4d

checked for the link [text](link) was intentional as it goes through markdown